app/assets/stylesheets/taxa/form.css

```diff
@@ -36,3 +36,35 @@ fieldset.column {margin-bottom: 1em;}
   width: 100%;
   height: 3em;
 }
+
+.jqmWindow {
+  display: none;
+  position: fixed;
+  top: 10%;
+  left: 50%;
+  margin-left: -350px;
+  width: 700px;
+  background-color: #EEE;
+  color: #333;
+  border: 1px solid black;
+  padding: 20px;
+}
+
+.jqmOverlay { background-color: #000; }
+
+* html .jqmWindow {
+  position: absolute;
+  top: expression( ( document.documentElement.scrollTop || document.body.scrollTop ) + Math.round( 17 * ( document.documentElement.offsetHeight || document.body.clientHeight ) / 100 ) + 'px' );
+}
+
+.potential_clashes {
+  margin-bottom: 10px;
+  display:  none;
+}
+
+td.fig20 {
+  width: 20%
+}
+td.fig10 {
+  width: 10%
+}
```

Comment:
Newline at the end of all files, please.

Reply:
done